frontend/src/pages/PostDetail/PostDetailPage.jsx

```diff
@@ -79,6 +86,7 @@ const PostDetailPage = ({ postToDisplay, commentsToDisplay, handleSearch, handle
         setModal={setModal}
         // eslint-disable-next-line no-underscore-dangle
```

Comment:
This rule seems like it conflicts with a bunch of code. If its needed and there's a few instances of such conflicts then perhaps it'd be better to overrite the rule in the `eslintrc.js` file.